src/v2/Apps/Order/Components/BankAccountPicker.tsx

```diff
@@ -27,6 +31,8 @@ export const BankAccountPicker: FC<Props> = props => {
     order,
     onSetPaymentSuccess,
     onSetPaymentError,
+    showInsuffiencyFundsError,
```

Comment:
minor: i would change all of these variables to be about insufficient* funds.

Reply:
After reading it a bit more i would name the variable something like `bankAccountHasInsufficientFunds` though i know that is a bit long.

Reply:
good catch! it's a typo, my intention was `showInsuffientFundsError`. I'll go with your suggestion as it's more descriptive 👍